src/theme/states/utils.ts

```diff
@@ -0,0 +1,21 @@
+import { colorShades } from '../palette';
+
+type GetShadeWithStepProps = {
+  shade: typeof colorShades[number] | 0;
+  step: number;
+};
+
+export const getShadeWithStep = ({
+  shade = 0,
+  step = 50,
+}: GetShadeWithStepProps): typeof colorShades[number] => {
+  let calculatedShade = shade;
+
+  if (shade + step > 950) {
```

Comment:
this 950 must be documented in another variable and use from there. This is used in filters as well and might be useful as we go